### PR TITLE
Add scripts

### PR DESCRIPTION
**Pull request by rodfersou**

### CODE REVIEW

File: src/sc/recipe/staticresources/bobtemplate/package.json.bob [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (hvelarde) [inline src/sc/recipe/staticresources/bobtemplate/package.json.bob]: what happens to this file when running buildout again? what happens to the options if I add or remove something manually?

File: src/sc/recipe/staticresources/bobtemplate/package.json.bob [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "main": "app/{{{short_name}}}.js",
+  "scripts": {
+    "build": "webpack -p",
+    "dev": "webpack --watch",
+    "watch": "webpack -p --watch"
+  },
+  "repository": {},
+  "license": "GPLv2",
+  "devDependencies": {
+    "babel-core": "^6.22.1",
```
Comment (hvelarde) [inline src/sc/recipe/staticresources/bobtemplate/package.json.bob]: you have to document all this features in the README file.

File: README.rst [hunk 3]
```diff
@@ -29,9 +29,74 @@ Don't Panic
 Installation
 ^^^^^^^^^^^^
 
-TBD.
+To enable this product in a buildout-based installation:
```
Comment (hvelarde) [inline README.rst]: before I install the product, I want to know what problems are solved by it and why should I use it; how this is going to make my life easier?

it should include a paragraph describing why we're using webpack.

File: setup.py [hunk 4]
```diff
@@ -41,6 +41,7 @@
       include_package_data=True,
       zip_safe=False,
       install_requires=[
+          'mr.bob',
```
[the remark this answers is not in the document]
Comment (rodfersou) [reply setup.py]: mr.bob does a lot more than just use Jinja2 template engine, like create file names based on parameter.  Also it enables our buildout to use custom templates from other packages.
Of course these features are not needed at first, but this way the integration with jinja is much simpler than to write code to handle all the template needs.

File: src/sc/recipe/staticresources/bobtemplate/package.json.bob (hunk 1 above)
Comment (rodfersou) [reply src/sc/recipe/staticresources/bobtemplate/package.json.bob]: Nothing happen.  This webpack [folder structure is created just if it don't exists](https://github.com/simplesconsultoria/sc.recipe.staticresources/pull/2/files#diff-7b7a2cd347045411db48e3650817a07dR85).

**Pull request merged**